pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -31,59 +31,108 @@ import (
 	"istio.io/istio/pilot/pkg/model"
 	"istio.io/istio/pilot/pkg/networking/plugin"
 	"istio.io/istio/pilot/pkg/networking/util"
-	"istio.io/istio/pilot/pkg/proxy/envoy/v1"
 	"istio.io/istio/pkg/log"
 )
 
-// Plugin is a mixer plugin.
-type Plugin struct{}
+type mixerplugin struct{}
```

Comment:
super nit: you could make this just one whole type block:

```
type (
        mixerplugin struct{}
        attribute   *mpb.Attributes_AttributeValue
        attributes  map[string]attribute
)
```

Reply:
I suggest leaving the type public since its methods are public, in which case it's more idiomatic to call it Plugin to avoid the stutter.